docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+##### max_tolerable_backend_down_num
+
+- **Unit**: -
+- **Default**: 0
+- **Description**: The maximum number of faulty BE nodes allowed. If this number is exceeded, Routine Load jobs cannot be automatically recovered.
+
+##### period_of_auto_resume_min
+
+- **Unit**: Min
+- **Default**: 5
+- **Description**: The interval at which Routine Load jobs are automatically recovered, in minutes.
+
+##### spark_load_default_timeout_second
+
+- **Unit**: s
+- **Default**: 86400
+- **Description**: The timeout duration for each Spark Load job, in seconds.
+
+##### spark_home_default_dir
+
+- **Unit**: -
+- **Default**: StarRocksFE.STARROCKS_HOME_DIR + "/lib/spark2x"
+- **Description**: The root directory of a Spark client.
+
+##### stream_load_default_timeout_second
+
+- **Unit**: s
+- **Default**: 600
+- **Description**: The default timeout duration for each Stream Load job, in seconds.
+
+##### max_stream_load_timeout_second
+
+- **Unit**: s
+- **Default**: 259200
+- **Description**: The maximum allowed timeout duration for a Stream Load job, in seconds.
+
+##### insert_load_default_timeout_second
+
+- **Unit**: s
+- **Default**: 3600
+- **Description**: The timeout duration for the INSERT INTO statement that is used to load data, in seconds.
+
+##### broker_load_default_timeout_second
+
+- **Unit**: s
+- **Default**: 14400
+- **Description**: The timeout duration for a Broker Load job, in seconds.
+
+##### min_bytes_per_broker_scanner
+
+- **Unit**: Byte
+- **Default**: 67108864
+- **Description**: The minimum allowed amount of data that can be processed by a Broker Load instance, in bytes.
+
+##### max_broker_concurrency
+
+- **Unit**: -
+- **Default**: 100
+- **Description**: The maximum number of concurrent instances for a Broker Load task. This parameter is deprecated from v3.1 onwards.
+
+##### export_max_bytes_per_be_per_task
+
+- **Unit**: Byte
+- **Default**: 268435456
+- **Description**: The maximum amount of data that can be exported from a single BE by a single data unload task, in bytes.
+
+##### export_running_job_num_limit
+
+- **Unit**: -
+- **Default**: 5
+- **Description**: The maximum number of data exporting tasks that can run in parallel.
+
+##### export_task_default_timeout_second
+
+- **Unit**: s
+- **Default**: 7200
+- **Description**: The timeout duration for a data exporting task, in seconds.
+
+##### empty_load_as_error
+
+- **Unit**: -
+- **Default**: TRUE
+- **Description**: Whether to return an error message "all partitions have no load data" if no data is loaded. Values: TRUE: If no data is loaded, the system displays a failure message and returns an error "all partitions have no load data". FALSE: If no data is loaded, the system displays a success message and returns OK, instead of an error.
```

Comment:
```suggestion
- **Description**: Whether to return an error message "all partitions have no load data" if no data is loaded. Values:
  - TRUE: If no data is loaded, the system displays a failure message and returns an error "all partitions have no load data".
  - FALSE: If no data is loaded, the system displays a success message and returns OK, instead of an error.
```